temba/channels/courier.py

```diff
@@ -39,7 +40,7 @@ def msg_as_task(msg):
     Used to serialize msgs as tasks to courier
     """
     msg_json = dict(id=msg.id,
-                    uuid=unicode(msg.uuid) if msg.uuid else "",
+                    uuid=text_str(msg.uuid) if msg.uuid else "",
```

Comment:
elsewhere we've started using six.text_type for this, tho it seems like it's always safe to use `str` for something like a UUID, no?

Reply:
Ah yeah I guess that makes sense for a UUID. I'm still very new to RapidPro so I might be being a bit overly cautious. Thanks!